src/dropzone.coffee

```diff
@@ -710,6 +710,11 @@ class Dropzone extends Emitter
             # Only the actual dropzone or the message element should trigger file selection
             if (clickableElement != @element) or (evt.target == @element or Dropzone.elementInside evt.target, @element.querySelector ".dz-message")
               @hiddenFileInput.click() # Forward the click
+          "touchstart": (evt) =>
+            evt.stopPropagation();        
```

Comment:
I should've commented on this line in source. Would it make sense to use `noPropagation(evt)` here?